charts/kubewarden-crds/Chart.yaml

```diff
@@ -22,9 +22,9 @@ keywords:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.4.5
+version: 1.5.0-rc3
```

Comment:
This should be

```suggestion
version: 1.4.6.0-rc1
```

Reply:
I'll apply this change and keep the pattern used in the other charts as well. I don't know why, but yesterday when I reviewed the previous commits I saw a minor version bump. Maybe my brain was failing in the end of the day... 

However,  it looks weird to me a rc for a patch version bump. Maybe is another issue to be fixed in the automation tech-debt issues.

Reply:
I think it's fine to do a version patch bump. The contents of the chart haven't changed, no features added, only chart annotations changed, hence a patch bump.